content/doc/installation/from-source.en-us.md

```diff
@@ -42,6 +42,12 @@ git tag -l
 git checkout v1.0.0
 ```
 
+If you want to build a `bindata` build, make sure to install the `go-bindata` dependency:
```

Comment:
Thougt about it, but I think it fits better here in the Download section than in the Build section below.